content/en/blog/2024/otel-collector-survey/index.md

```diff
@@ -0,0 +1,93 @@
+
+### Customization and Configuration
+A surprising number of people build their own distributions of the Collector (61/186), indicating that providing a composable Collector is important for the community. Most people who do build their own Collector distros use the [OTel Collector Builder (OCB)](https://github.com/open-telemetry/opentelemetry-collector-builder) (49/61). Of the 49 respondents who leverage the OCB, most were able to figure it out, with only 2 respondents stating that it’s too hard to use.
+
+![Chart showing how easy people find it to use the otel collector builder](ocb-usage.png)
+
+### Monitoring and Observability
+When it comes to monitoring the Collectors, a vast majority of respondents rely on Collector metrics and logs (70%), and only a few don't monitor their Collector(s) at all (16%). When we peered deeper into the data, we found that of the 125 respondents with more than 5 Collectors, only 15 are not monitoring their Collectors, and of the 100 respondents with more than 10 Collectors, only 9 are not monitoring their Collectors. This seems to indicate that users take monitoring their Collectors seriously once they reach a certain maturity with their Collector deployments.
+
+![Chart showing how people monitor their otel collector](monitoring.png)
+
+### OTel Components Usage
+The OTel Collector's flexibility is vividly illustrated by the wide array of exporters, receivers, processors, connectors, and extensions that are used across various environments. This highlights the Collector's capability to integrate with a vast range of tools and systems.
+
+The top components according to our survey results are as follows:
+
+#### Receivers
+- OTLP Receiver
+- Prometheus Receiver
+- File Receiver
+
+#### Exporters
+- OTLP Exporter
+- Prometheus Exporter
+
+#### Processors
+- Batch Processor
+- Attributes Processor
+- Filter Processor
+- Memory Limiter Processor
+- K8s Attribute Processor
+
+#### Connectors
+- Span Metrics Connector
+- Servicegraph Connector
+- Count Connector
+- Routing Connector
+
+#### Extensions
+- Health Check Extension
+- Basic Auth Extension
+- Pprof Extension
+- Bearer Token Auth Extension
+- Storage Extension
+
+For a more detailed look at the specific exporters, receivers, processors, connectors, and extensions in use, you can check out the [raw results](https://github.com/open-telemetry/sig-end-user/blob/main/end-user-surveys/otel-collector/otel-collector-survey.csv). The data provides a clear view of the popular choices within the community as well as the niche configurations that exemplify the customizability of the OTel Collector.
+
+### Areas for Improvement
+Respondents were clear about their desire for enhancements in stability (30.6%),  configuration management and resolution (30.1%), and self-observability (28%), versus wanting new components (<8%)
+
+![Chart showing interested areas of improvement for the otel collector](areas-of-improvement.png)
```

Comment:
Same comment as with a previous graph:  there are 7 items with 1 response, taking over the space for the three relevant answers (the first three). So, grouping them in an "Others" makes sense.

Reply:
Done

Reply:
can this conversation be marked as resolved?